<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-U8N41C8" failures="1" tests="6" name="gra.Tests.GraTestWithFakerAndPropertyFile" time="9.949" errors="0" timestamp="2023-01-16T20:50:01 IST" skipped="0">
  <testcase classname="gra.Tests.GraTestWithFakerAndPropertyFile" name="CreateRepoAPITest" time="6.255"/>
  <system-out/>
  <testcase classname="gra.Tests.GraTestWithFakerAndPropertyFile" name="GetRepoAPITest" time="0.495">
    <failure message="expected [200] but found [404]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:1240)
at org.testng.Assert.assertEquals(Assert.java:1274)
at gra.Tests.GraTestWithFakerAndPropertyFile.GetRepoAPITest(GraTestWithFakerAndPropertyFile.java:55)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
    </failure>
  </testcase> <!-- GetRepoAPITest -->
  <system-out/>
  <testcase classname="gra.Tests.GraTestWithFakerAndPropertyFile" name="UpdateRepoAPITest" time="0.944"/>
  <system-out/>
  <testcase classname="gra.Tests.GraTestWithFakerAndPropertyFile" name="DeleteRepoNoAdminAPITest" time="0.458"/>
  <system-out/>
  <testcase classname="gra.Tests.GraTestWithFakerAndPropertyFile" name="DeleteRepoWithAdminAPITest" time="1.365"/>
  <system-out/>
  <testcase classname="gra.Tests.GraTestWithFakerAndPropertyFile" name="GetRepoAPITestAfterDelete" time="0.432"/>
  <system-out/>
</testsuite> <!-- gra.Tests.GraTestWithFakerAndPropertyFile -->
